deepspeed/runtime/utils.py

```diff
@@ -856,3 +856,14 @@ def call_to_str(base, *args, **kwargs):
         name += ', '.join(f'{key}={repr(arg)}' for key, arg in kwargs.items())
     name += ')'
     return name
+
+def print_backward_tensors(grad_fn):
```

Comment:
How about this function take a tensor and extract the grad_fn if available or return immediately otherwise? This could make it easier for users, especially if torch was to change the grad_fn attribute in future :).

Reply:
cool, will change it this way, thanks!